### DISCUSSION TITLE
GittHub Mobile

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#37907] [Mobile] [answered]
Can't we review the code on our GitHub Mobile app?

**Comment by Iron-Ham** [comment, marked as answer]
Hi @iLakshya,

Yes, you can!
To review code, first navigate to the pull request you want to review:
![Simulator Screen Shot - iPhone Xʀ - 2022-10-31 at 15 07 31](https://user-images.githubusercontent.com/3388381/199089790-e60b5a0d-05e1-4437-af57-d45f5cf9f581.png)

Then, tap on the cell that indicates how many files have changed: 
![Simulator Screen Shot - iPhone Xʀ - 2022-10-31 at 15 07 57](https://user-images.githubusercontent.com/3388381/199090002-cf7f91c7-5571-44d5-b1bc-e7e015ca90a6.png)

From there, you can leave comments on individual lines by tapping those lines, and you can also select multiple lines to comment on:

| Comment on a Single Line | Selecting multiple line to comment on |
|--------|--------|
| ![Simulator Screen Shot - iPhone Xʀ - 2022-10-31 at 15 09 53](https://user-images.githubusercontent.com/3388381/199090210-eeb5e37d-7e5f-4211-9ee7-fa75196e3b84.png) | ![Simulator Screen Shot - iPhone Xʀ - 2022-10-31 at 15 09 39](https://user-images.githubusercontent.com/3388381/199090178-6b2bd09d-e5be-4a91-95fe-8635fd73eeb2.png) | 

Once you're done leaving your comments, tap the `Review Changes` button to submit your final review and mark it as an `Approval`, `Request Changes` or `Leave a comment`. Note that in this example, I am reviewing my own pull request and thus the only option available to me is `Leave a comment`. 

![Simulator Screen Shot - iPhone Xʀ - 2022-10-31 at 15 12 05](https://user-images.githubusercontent.com/3388381/199090577-30e1eba1-c4de-43b0-b216-9c69be86f29c.png)

I hope this was helpful!
